doc/sphinx/cti/input-files.rst

```diff
@@ -709,4 +709,3 @@ in the report for Chemkin referenced above. These errors include:
 
 .. [SAND98] See R. J. Kee, G. Dixon-Lewis, J. Warnatz, M. E. Coltrin, J. A. Miller,
    H. K. Moffat, Sandia National Laboratories Report SAND86-8246B (1998).
-   
```

Comment:
@EmilAtz Let us know if you want to fix it, if not, this should be good to merge.